assets/src/application/ui/display/viewport/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as ifViewportMatches } from './if-viewport-matches';
+export { default as withViewportMatch } from './with-viewport-match';
```

Comment:
the reason why this package was added is because of this error:
![image](https://user-images.githubusercontent.com/1477453/77543994-19d6dd80-6eb1-11ea-99e2-08fbff4fc964.png)
***

I removed the code that wasn't needed and was causing the problems